### DISCUSSION TITLE
Enclosure v3 STL

### DISCUSSION THREAD

**Discussion by liamamiller** [CircuitSetup/Time-Circuits-Display#36] [Q&A]
Is there any way the V3 STL files for the left and right enclosure files could be uploaded? I'm attempting to make a stand but want to line up the rear mounting holes properly. Think the STL files would be a good template to make the stand. Thanks!

**Comment by CircuitSetup** [comment]
They ate already available here: https://github.com/CircuitSetup/Time-Circuits-Display/tree/master/Hardware/3d%20Files
😀
